.github/workflows/scripts.yml

```diff
@@ -46,15 +46,22 @@ jobs:
       matrix:
         globalnet: ['', 'globalnet']
         deploytool: ['operator', 'helm']
+        cable_driver: ['libreswan', 'strongswan', 'wireguard']
```

Comment:
This shouldn't be here as well.

Reply:
Should I remove the globalnet tests too then, to be consistent with this reasoning?